### PR TITLE
Accessibility Updates (Screenreader, Semantic HTML, Keyboard)

### PR DESCRIPTION
**Pull request by DynieM**

### CODE REVIEW

**Review by clpetersonucf — CHANGES_REQUESTED** [review]
Nice work! Only a few small things I've noticed:

- The previous and next buttons are both focusable and selectable when there aren't previous or next pages to paginate to. Ideally, these buttons are disabled (they should change visually as well as being non-interactive). Additionally, the aria labels associated with these buttons don't appear to update the current page's progress correctly.
- The keyboard instructions dialog doesn't appear to be interactive at all with the screenreader. Honestly, that instructions pop-up is super dated anyways. I believe I'd rather see it turned into a proper modal dialog, with the corresponding accessibility attributes, and the information about how to play could likely be condensed and simplified. The visual style and presentation is up to you - the "page of an instruction booklet" style it has now is cute, but it also makes it more difficult to comprehend at a glance.

**Review by cayb0rg — CHANGES_REQUESTED** [review]
This looks great, Dee! There's just a couple minor issues I noticed:

- (More of a nitpick) The `_assistiveNotification` and `_assistiveAlert` both appear to do the same thing, so we should probably switch to using one or the other.
- The close button for the instructions modal is still tabbable when closed. This might be due to using `tabindex=1`, which could be fixed by using the `focus()` function instead when the instruction modal is opened to focus the close button. 
- The game board is still tabbable when the instructions are opened. 

Also, I would maybe suggest adding the keyboard instructions as an `aria-label` to the Help button to make finding them easier, or as an aria-label to the instruction modal's close button.

Overall, this looks good. Nice work!

File: src/controllers/player.coffee [hunk 1]
```diff
@@ -159,13 +176,32 @@ Matching.controller 'matchingPlayerCtrl', ['$scope', '$timeout', '$sce', ($scope
...
+		else if direction == 'previous' then _assistiveNotification 'Page decremented. You are on the previous page.'
+
+
+	_updateCompletionStatus = () ->
+		$scope.completePerPage  = []
+		for match in $scope.matches
+			if !$scope.completePerPage[match.matchPageId] then $scope.completePerPage[match.matchPageId] = 1
+			else $scope.completePerPage[match.matchPageId]++
+		console.log($scope.completePerPage)
+		console.log($scope.currentPage)
```
Comment (cayb0rg) [inline src/controllers/player.coffee:190]: Just a reminder to clear any leftover console.logs

**Comment by clpetersonucf** [comment]
This appears to be superseded by #40, closing

**Pull request closed without merging**